src/app_service/service/chat/service_interface.nim

```diff
@@ -0,0 +1,98 @@
+import json
```

Comment:
This file is not needed anymore, you can delete it. The Service is a QtObject

Reply:
@borismelnik I am not sure where does this `service_interface` come from, since we removed it some time ago and it's not a part of `base_bc`. Most likely this is a product of some rebase or whatever. Please remove it.